src/main/java/com/mulberry/blackmulberry/controller/main/AccountController.java

```diff
@@ -2,13 +2,14 @@
 
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 
+@RequestMapping("account")
```

Comment:
cool that you add request mapping, but i think 'account' url now is in locale : localhost:8080account, probebly you need add / for its be '/account' -> localhost:8080/account